src/index.js

```diff
@@ -64,3 +65,10 @@ form.addEventListener('submit', (e) => {
 document.getElementById('refresh').addEventListener('click', () => {
   getData();
 });
+
+// const getScores = async (cont) => {
+//   const response = await fetch(scoresUrl);
+//   const jsonData = await response.json();
+//   const sortArr = jsonData.result.sort((a, b) => b.score - a.score);
+//   frameCreation(sortArr, cont);
+// };
```

Comment:
- [ ] OPTIONAL
There are comments found in your code on the repository. Kindly endeavor to remove **all** comments from your code before pushing it to the repository so as to keep your code cleaner.

Reply:
okay I'll remove that.